### DISCUSSION TITLE
[SOLVED] - Only domain recorded as referrer

### DISCUSSION THREAD

**Discussion by jeffshead** [YOURLS/YOURLS#3490] [Q&A] [answered]
When I view the recorded referrers in the log, they only shows the domain.

For example, if I place a short link on: `https://www.example.com/directory/subdirectory/index.php`, and someone clicks that link, the referrer recorded in the log will be: `https://www.example.com/`

Shouldn't the referrer be:  `https://www.example.com/directory/subdirectory/` or `https://www.example.com/directory/subdirectory/index.php`

Is this by design?

**Comment by dgw** [comment, marked as answer]
Browsers are shifting/have shifted to a stricter default [referrer policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy#directives). While the old default was `no-referrer-when-downgrade`, cross-origin requests will likely only get the origin (domain) by default from most browsers now as the `strict-origin-when-cross-origin` policy has been adopted as the new default by browsers including Firefox and Chrome/Chromium.

YOURLS only caps the length of the referrer, though a plugin could further modify it:

https://github.com/YOURLS/YOURLS/blob/2f3ea603ae8e0c8c5068ea1de5e3676352649946/includes/functions.php#L220-L224
